index.js

```diff
@@ -1,10 +1,74 @@
 'use strict';
 const crypto = require('crypto');
 
-module.exports = length => {
+const urlSafeChars = 'abcdefjhijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-._~'.split('');
+
+const generateForCustomChars = (length, chars) => {
```

Comment:
Can you add a comment that describes what this does and how it does it?

Reply:
done